src/pages/campaign/NewCampaign.tsx

```
@@ -76,80 +91,97 @@
-    });
+    const twoMinutesFromNow = Math.floor((Date.now() + 2 * 60 * 1000) / 1000);
+
+    try {
+      await submitCampaign(twoMinutesFromNow);
+
+      toast({
+        title: 'Campaign Sent',
+        description: 'Your campaign has been sent successfully.',
+      });
+      resetForm();
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)